components/Footer.js

```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
+			<div className='container p-6 mx-auto lg:space-y-8'>
+				<h2 className='text-4xl font-lexend font-bold text-center'>
+					Let&#39;s connect!
+				</h2>
+				<div className='flex flex-wrap justify-center justify-around px-80'>
```

Comment:
What is the `justify-around` and `px-80` for? `px-80` is a dangerously large margin value and breaks on mobile, where the icons aren't centered: 
<img width="571" alt="image" src="https://user-images.githubusercontent.com/36285610/184803367-2091495b-d932-44a2-876f-2ccc30cab65a.png">